src/CsvTextEditor/Commands/FileOpenInTextEditorCommandContainer.cs

```diff
@@ -19,5 +22,29 @@ public class FileOpenInTextEditorCommandContainer : FileOpenInExternalToolComman
             : base(Commands.File.OpenInTextEditor , "txt", commandManager, projectManager, fileExtensionService, fileService, processService)
         {
         }
+
+
+        protected override void Execute(object parameter)
+        {
+
+            var resolver = ServiceLocator.Default.GetDependencyResolver();
+            var processService = resolver.Resolve<IProcessService>();
```

Comment:
It's better to inject the services into the ctor, the dependency resolver will automatically take care of that for you.